docs/source/examples/diffraction_objects_example.rst

```diff
@@ -180,7 +180,7 @@ For example, attempting to add a diffraction object and a string will raise an e
 
 .. code-block:: python
 
-    tth_ninety_index = diff_object1.get_array_index(90, xtype="tth")
+    tth_ninety_index = diff_object1.get_array_index(xvalue=90, xtype="tth")
```

Comment:
This will have to be fixed later when we finalize the signature.  Let's just remember to do it....